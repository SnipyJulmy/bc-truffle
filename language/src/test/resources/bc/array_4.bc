a[12] = 213
a[12]
define get(a[],i) {
  return a[i];
}
a[12]
get(a[],12)
a[1]
get(a[],1)
halt
