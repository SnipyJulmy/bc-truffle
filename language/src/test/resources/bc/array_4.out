213
213
213
0
0
